src/interfaces/index.ts

```diff
@@ -619,6 +619,12 @@ export interface INewLivechatRoomCredentialAPI {
   agentId?: string
 }
 
+/** Structure to get livechat message */
+export interface ILivechatMessageCredentialAPI {
+  rid: string
+  token: string
```

Comment:
I think we don't need the `token` property here, because you are already sending it through the request. :thinking: